tests/integration/ha_tests/conftest.py

```diff
@@ -7,7 +7,8 @@
 from pytest_operator.plugin import OpsTest
 from tenacity import Retrying, stop_after_delay, wait_fixed
 
-from tests.integration.ha_tests.helpers import (
+from ..helpers import run_command_on_unit
+from .helpers import (
```

Comment:
Relative imports seems to be the easiest way to get the test collector to play game.